docs/source/enable_cc_lifecycle.md

```diff
@@ -0,0 +1,300 @@
+			           "rules": [
+			           	{
+			           		"signed_by": 0
+			           	}
+			           ]
+			       }
+              	},
+              	"version": 0
+              }
+           },
+           "version": "0"
+      }
+   },
+   "appPolicies": {
+ 		"Endorsement": {
+			"mod_policy": "Admins",
+			"policy": {
+				"type": 3,
+				"value": {
+					"rule": "MAJORITY",
+					"sub_policy": "Endorsement"
+				}
+			},
+			"version": "0"
+		},
+		"LifecycleEndorsement": {
+			"mod_policy": "Admins",
+			"policy": {
+				"type": 3,
+				"value": {
+					"rule": "MAJORITY",
+					"sub_policy": "Endorsement"
+				}
+			},
+			"version": "0"
+		}
+   },
+   "acls": {
+		"_lifecycle/CommitChaincodeDefinition": {
+			"policy_ref": "/Channel/Application/Writers"
+		},
+		"_lifecycle/QueryChaincodeDefinition": {
+			"policy_ref": "/Channel/Application/Readers"
+		},
+		"_lifecycle/QueryNamespaceDefinitions": {
+			"policy_ref": "/Channel/Application/Readers"
+		}
+   }
+}
+```
+
+## Edit the channel configurations
+
+### System channel updates
+
+Because configuration changes to the system channel only involve parameters
+inside the configuration of the individual ordering service organizations within
+the channel configuration, the relevant organizations can make the edits without
+needing additional signatures.
+
+You will need to export the following variables:
+
+* `CH_NAME`: the name of the system channel being updated.
+* `CORE_PEER_LOCALMSPID`: the MSP ID of the organization proposing the channel
+  update. This will be the MSP of one of the orderer organizations.
+* `ORGNAME`: The name of the organization you are currently updating.
+* `CORE_PEER_TLS_ROOTCERT_FILE`: the absolute path to the TLS cert of your
+  ordering node(s). Need clarification here.
+* `CORE_PEER_MSPCONFIGPATH`: the absolute path to the MSP representing your
+  organization.
+* `ORDERER_CA`: the absolute path to the root CA certificate of the organization
+  proposing the system channel update.
+* `ORDERING_NODE_ADDRESS`: the url of an ordering node.
+  When targeting the ordering service, you can target any particular node in the
+  ordering service. Your requests will be forwarded to the leader automatically.
+
+Once you have set the environment variables, navigate to
+[Step 1: Pull and translate the config](./config_update.html#Step-:-Pull-and-translate-the-config).
+
+Once you have a `modified_config.json`, add the lifecycle organization policy
+(as listed in `enable_lifecycle.json`) using this command:
+
+```
+jq -s ".[0] * {\"channel_group\":{\"groups\":{\"Consortiums\":{\"groups\": {\"FabricConsortium\": {\"groups\": {\"$ORGNAME\": {\"policies\": .[1].${ORGNAME}Policies}}}}}}}}" config.json ./enable_lifecycle.json > modified_config.json
+```
+
+Then, follow the steps at [Step 3: Re-encode and submit the config](./config_update.html#Step-3-Re-encode-and-submit-the-config).
+
+If every ordering service organization performs their own channel edit, they can
```

Comment:
It is `non-orderer org` not `ordering service org`. Also, it is better to mention that this should be done for all orgs, something like `The above update needs to be performed to all the organizations in a consortium. If every organization performs their own channel edit,`.

Reply:
As with the above comment, I've fixed this to reflect the process involved here.